app/components/error_list/connection_error.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {
+    StyleSheet,
+    View,
+    Text
+} from 'react-native';
+
+const style = StyleSheet.create({
+    container: {
+        alignSelf: 'stretch',
+        alignItems: 'center',
+        justifyContent: 'center',
+        padding: 10
+    }
+});
+
+function ConnectionError() {
+    return (
+        <View style={style.container}>
+            <Text>{'I am a general error'}</Text>
```

Comment:
this should be localized.

@lfbrock can you add the text that needs to be displayed? My understanding is that this message appears when a connection error has occurred.